cockatrice/src/carditem.cpp

```diff
@@ -16,6 +17,7 @@
 #include "settingscache.h"
 #include "tab_game.h"
 #include "pb/serverinfo_card.pb.h"
+#include "abstractcarditem.h"
```

Comment:
They were already there, but I can do some experiments and see if they're really needed.